ivy/functional/frontends/torch/linalg.py

```diff
@@ -370,3 +370,16 @@ def cholesky_ex(input, *, upper=False, check_errors=False, out=None):
             matrix = input * math.nan
             info = ivy.ones(input.shape[:-2], dtype=ivy.int32)
             return matrix, info
+
+
+def lu_factor_ex(A, *, pivot=True, check_errors=False, out=None):
+    try:
+        LU, pivots = ivy.lu_factor(A, pivot=pivot, out=out)
+        info = ivy.zeros(A.shape[:2], dtype=ivy.int32)
+        return LU, pivots, info
+    except RuntimeError as e:
+        if check_errors:
+            raise RuntimeError(e)
+        else:
+            LU = A * math.nan
+            info = ivy.ones(A.shape[:2], dtype=ivy.int32)
```

Comment:
Hi @kurshakuz , I am trying to understand what `lu_factor_ex` should look like. I read the pytorch documentation but I am not getting what we should do under this else case, I mean what should be the LU, pivots, and info values here? Can you please help me to understand it?

Reply:
Hey! I have not worked on that function implementation as well. Would you mind rising that question in the Torch Discussion website? I believe someone with more experience can help. Otherwise, you are also welcome to raise that question in the Discord frontends channel!